content/docs/command-reference/import.md

```diff
@@ -137,6 +137,39 @@ Several of the values above are pulled from the original stage file
 subfields under `repo` are used to save the origin and version of the
 dependency, respectively.
 
+### Importing from non-DVC repositories
+
+You can even import files (datasets) from Git repositories that are not
+specifically <abbr>DVC repositories</abbr>. Here's an example importing a
+dataset from [GSA's data repo](https://github.com/GSA/data).
+
+```dvc
+$ dvc import git@github.com:GSA/data \
+           enterprise-architecture/it-standards.csv
+Importing 'enterprise-architecture/it-standards.csv
```

Comment:
minor: is the output actually split? if not - let's keep it as-is, for output we tend to preserve it as we have it in DVC

Reply:
Okay, but that makes the code div scrollable. Is that okay?

Reply:
yes, it's fine ... it's better to keep it close to the actual output

Reply:
Hm I would personally prefer non-scrollable code/cli samples. No right or wrong answer here but for consistency should we go for non-scrolling? Most command/cli samples I've seen try to avoid the horizontal scrolling (that means up to 72 character-long lines).

Reply:
there are two types of lines:

1. **commands** - start with `$`. It's reasonable to avoid the horizontal scroll. There is a mechanism for that - `\`.
2. **output** - this is one is not that simple. If we start splitting output w/o any special sign that we wrapped it there is a danger at some point that we can get a different semantic for it. If it looks like it's too much I would try simplify with `...` instead of something like that, but not silently introduce newlines that are actually not there.

Reply:
Good points, I use `...` often to shorten these output lines. No need to keep the whole thing, just give an idea of what to expect from running commands and only when absolutely needed (best to avoid sample command output altogether). Please note, @utkarshsingh99